target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -138,7 +138,9 @@ case "$FIRMWARE" in
 	compex,wpj428 |\
 	engenius,eap1300 |\
 	openmesh,a42 |\
-	openmesh,a62)
+	openmesh,a62 \|
```

Comment:
What happend here? Shouldn't this be "|\"?

Reply:
i'm so sorry.It's my mistake.